js/App.js

```diff
@@ -125,11 +126,43 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits = 8) {
   return new Intl.NumberFormat(window.lang, {maximumFractionDigits: maxDigits}).format(numberToFormat);
 };
 
+App.prototype.getBitcoinUnit = function() {
+  if (!this._bitcoinUnit) {
+    this._bitcoinUnit = localStorage.getItem('BitcoinUnit') || 'BTC';
+  }
+  return this._bitcoinUnit;
+};
+
+App.prototype.setBitcoinUnit = function(unit) {
+  this._bitcoinUnit = unit;
+  localStorage.setItem('BitcoinUnit', unit);
+};
+
+/**
+ * Format a bitcoin amount in the user's locale.
+ *
+ * @param {Number} amount - Bitcoin (BTC) amount.
+ * @param {Number} [maxDigits=8] - Maximum number of fraction digits to display.
+ * @returns {String} localised amount ending with ' BTC' or the default bitcoin unit.
+ *
+ * @see intlNumFormat
+ */
+App.prototype.formatBitcoin = function(amount, maxDigits) {
```

Comment:
Nan as money, interesting concept.  I'll fix the code.

I tested everything I could.  I don't have a real store, just supported the effort because I'm interested in distributed computing and crypto currencies. 

Is there a database I can copy to get transactions and refunds?  How do you test the code?